src/sentry/api/serializers/models/repository_project_path_config.py

```diff
@@ -6,16 +6,21 @@
 @register(RepositoryProjectPathConfig)
 class RepositoryProjectPathConfigSerializer(Serializer):
     def serialize(self, obj, attrs, user):
-        integration = obj.organization_integration.integration
-        provider = integration.get_provider()
```

Comment:
@NisanthanNanthakumar Not sure if I mentioned this but do we need to update the Typescript types for this now that the serializer is changing?

Reply:
yes! good catch!